.circleci/config.yml

```diff
@@ -45,13 +71,39 @@ jobs:
             pip3 install coveralls
             python3 setup.py test -P visualization --remote-data=astropy -a "--mpl"
             coveralls || true
+  html-docs:
+    docker:
+      - image: circleci/python:3.6
+    steps:
+      - checkout
+      - run: *apt-install
+      - run: *pip-install
+      - run: venv/bin/python setup.py build_docs -w
+
+      - store_artifacts:
+          path: docs/_build/html
+
+      - run:
+          name: "Built documentation is available at:"
+          command: DOCS_URL="${CIRCLE_BUILD_URL}/artifacts/${CIRCLE_NODE_INDEX}/${CIRCLE_WORKING_DIRECTORY/#\~/$HOME}/docs/_build/html/index.html"; echo $DOCS_URL
 
 
 workflows:
   version: 2
+
+  egg-info:
+    jobs:
+      - egg-info-36
+      - egg-info-35
+
   build_and_test:
     jobs:
+      - html-docs
       - 32bit
       - image-tests-mpl153
       - image-tests-mpl202
       - image-tests-mpl212
+
+notify:
+  webhooks:
+    - url: https://giles.cadair.com/circleci
```

Comment:
if you go the astropy-bot route this will need updating with the url of the astropy-bot

Reply:
This https://github.com/astropy/astropy-bot/pull/91 is a prerequisite, then it should be easy enough to add the giles bit into astropy bot.

Either way the webhook is only one tiny part of this PR.